captchasrv/captchasrv.pl

```diff
@@ -36,7 +36,8 @@
 #    <IMG SRC="http://host:port/path/capP5E6VcbV.jpg" ALT="CAPTCHA image">
 
 #use strict; # Clashes with constant definitions
-use vars qw($opt_t $opt_o $opt_h $opt_p); # For command-line options with Getopt
+use vars qw($opt_t $opt_o $opt_h $opt_p)
+    ;    # For command-line options with Getopt
```

Comment:
To avoid the reformatted comment being effectively after the commented line, we might have to move all these comments to their own line before the commented statement,